Makefile.toml

```diff
@@ -138,7 +138,7 @@ category = "Build"
 description = "Compiles for all targets needed to produce a universal library for watchOS and tvOS"
 command = "rustup"
 args = [
-    "run", "${CARGO_MAKE_TOOLCHAIN}",
+    "run", "nightly", #nightly needed for -Z build-std 
```

Comment:
I don't have a better solution than this but there is a scenario where it's nice to set a local build of rustc. Hopefully watchOS/visionOS/tvOS targets don't break anytime soon but they have in the past.